### PR TITLE
Use JSON schema for subject, visit, observation

### PR DESCRIPTION
**Pull request by abhidg**
These JSON schemas can be used to validate mappings.

There are some changes from schemas.py -- I am using dataset_id which will be filled by a future ingestion pipeline, also dropped some HIV measurements (latest CD4) from subject, these should be in observations.

### CODE REVIEW

File: isaric/parsers/isaric-ccpuk.toml [hunk 1]
```diff
@@ -428,15 +428,3 @@
     { field = "pronevent_prtrt", ref = "Y/N/NK", description = "Prone ventilation" },
     { field = "daily_prone_cmtrt", ref = "Y/N/NK", description = "Prone positioning" },
   ]
-
-[visit.treatment_tocilizumab] # TODO: antiviral_cmtrt == 9 condition, but multiple fields, needs to be fixed
```
Comment (pipliggins) [inline isaric/parsers/isaric-ccpuk.toml:432]: Any reason these two were deleted?

**Review by pipliggins — APPROVED** [review]
Looks good to me! Having the parsers explicit checked against the schema will be useful.

File: isaric/parsers/isaric-ccpuk.toml (hunk 1 above)
Comment (ekamau) [reply isaric/parsers/isaric-ccpuk.toml:432]: I am wondering the same ..

**Review by ekamau — APPROVED** [review]
Looks good.

File: isaric/parsers/isaric-ccpuk.toml (hunk 1 above)
Comment (abhidg) [reply isaric/parsers/isaric-ccpuk.toml:432]: Was giving validation errors, added back in with corrected field names.

**Pull request merged**